ui-modules/catalog/app/views/catalog/catalog.template.html

```diff
@@ -69,7 +69,7 @@
         </div>
 
         <div class="col-md-12">
-            <ul class="list-group list-group-bundles" ng-show="state.view === 'bundles'">
+            <ul class="list-group list-group-bundles" ng-if="state.view === 'bundles'">
```

Comment:
big savings -- previously would create DOM for every single entity, even though none shown initially